tsconfig.json

```diff
@@ -14,7 +14,10 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "react-jsx"
+    "jsx": "react-jsx",
+    "suppressImplicitAnyIndexErrors": true,
```

Comment:
```suggestion
```

Nope ^^. If you have to disable this kind of error, it seems there is a problem with your types to fix.